tests/testUtils.py

```diff
@@ -244,18 +245,18 @@ def writeCSV(self, data, fileName):
 
     {'id1' : {'f1':n1, 'f2':n2}, 'id2' : {'f1':n3, 'f2':n4}}
     """
-    csvFile = open(fileName, 'wb')
+    csvFile = open(fileName, 'w')
     csvFileWriter = csv.writer(csvFile)
     # get the headers from the first row
-    header = sorted(data[data.keys()[0]].keys())
+    header = list(data[list(data.keys())[0]].keys())
```

Comment:
@jcfr I changed this as I forgot to add the feature class to the output (meaning the features from different classes ended up mixed together due to the sorting). However, the output now again contains also the feature classes, meaning applying the sorted() function nicely sorts the features alphabetically per featureclass (as feature classes are also sorted alphabetically). So in this case, I'd propose to revert this to `sorted(data[list(data.keys())[0]].keys())`